README.md

```diff
@@ -1,10 +1,11 @@
-# Howso Engine&trade;
 
-The Howso Engine is an application that enables data exploration and machine learning capabilities through
-the creation and use of Trainees that help users store, explore, and analyze the relationships in their data.
-The Howso Engine utilizes a instance-based learning approach with strong ties to the [k-nearest neighbors algorithm](https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm) and [information theory](https://en.wikipedia.org/wiki/Information_theory).
+<div align="left">
+  <img src="https://docs.howso.com/_static/howso_logo_icon.svg">
```

Comment:
Can't tell in GH, but @mkoistinen had mentioned that the Howso logo with a non-white background looked goofy in VSCode dark mode. If we're going forward with the white background in other repos, I'd make sure this one has it too.

Reply:
I don't want to mess with putting a local png file in this repo, and the docs site doesn't have the white background one here... So I think this can wait until day 2 or so